packages/prisma/package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "scripts": {
     "clean": "rm -rf .turbo && rm -rf node_modules",
-    "build": "yarn prisma migrate deploy || true",
+    "build": "yarn prisma migrate deploy && yarn post-build || true",
+    "post-build": "yarn prisma db execute --file scripts/post-migration.sql",
```

Comment:
Does this mean that if you set the env variable it will run the script every single time migrations are run? 

In our case, as you mentioned in the description, it’s only necessary sometimes. I think this should take that in to account.

Reply:
I think that would make the logic unnecessarily more complex as we would need to check if the schema changed in a way that a new column was added.

Running the script multiple times doesn't yield side effect if there's no new tables are added

Reply:
It’s just heavy-handed to grant all new permissions on all tables in the database for no reason when there aren’t new tables added. I like the idea of automating this but I really feel this isn’t the correct approach.

Reply:
I'm open to suggestions. We could orchestrate all of this from gh actions themselves. Although that would increase the chance of having a larger timeframe where the DB is de-synced from the code. Just trying to be a close to the metal as possible.

Reply:
We write a trigger that runs when create table occurs in the db and we run this script then and only then.